app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryShowActivity.kt

```diff
@@ -78,6 +91,15 @@ class ItineraryShowActivity : AppCompatActivity() {
         startActivity(intent)
     }
 
+    /**
+     * Delete this mapping mission and go back to the mission selection activity
+     */
+    fun deleteMission(@Suppress("UNUSED_PARAMETER") view: View) {
+        mappingMissionDao.removeMappingMission(ownerUid, privateId, sharedId)
+        val intent = Intent(this, MappingMissionSelectionActivity::class.java)
```

Comment:
It would be nice to add a dialog to make sure the user wants to delete the mission, there is a useful class for that and the issue was addressed here: https://stackoverflow.com/questions/2115758/how-do-i-display-an-alert-dialog-on-android

Reply:
Je viens de l'ajouter si tu veux vérifier